pkg/rhelv2/rpm/rpm.go

```diff
@@ -17,42 +17,47 @@ import (
 	"github.com/stackrox/scanner/ext/featurens/redhatrelease"
 	"github.com/stackrox/scanner/pkg/analyzer"
 	"github.com/stackrox/scanner/pkg/metrics"
-	"github.com/stackrox/scanner/pkg/repo2cpe"
 	"github.com/stackrox/scanner/pkg/rpm"
 )
 
 const (
-	contentManifests = `root/buildinfo/content_manifests`
-
 	pkgFmt = `rpmv2`
 )
 
 // AllRHELRequiredFiles lists all the names of the files required to identify RHEL-based releases.
 var AllRHELRequiredFiles set.StringSet
 
+// contentManifests set with all known directories that might contain content manifest files.
+var contentManifestDirs set.StringSet
```

Comment:
```suggestion
var (
	// AllRHELRequiredFiles lists all the names of the files required to identify RHEL-based releases.
	var AllRHELRequiredFiles set.StringSet

	// contentManifests set with all known directories that might contain content manifest files.
	var contentManifestDirs set.StringSet
)
```

Reply:
also nit: comment name should start with the variable's name

Reply:
See 0ee22232